spyder/plugins/ipythonconsole/utils/kernelspec.py

```diff
@@ -117,27 +117,35 @@ def argv(self):
                 self.set_conf('default', True, section='main_interpreter')
                 self.set_conf('custom', False, section='main_interpreter')
 
+        kernel_cmd = []
+
+        if is_conda_env(pyexec=pyexec):
+            # If executable is a conda environment and different from Spyder's
+            # runtime environment, use conda's "run" subcommand in order to
+            # activate the environment and run spyder-kernels.
```

Comment:
```suggestion
            # If executable is a conda environment, use "run" subcommand to
            # activate it and run spyder-kernels.
```

Update comment to reflect what's happening here now.